### ISSUE TITLE
Video content tabs

### ISSUE THREAD

**Issue by ChaykeeMor** [ji-devs/ji-cloud#3439]
The icons in these inactive tabs should be faded like it's corresponding text.
BUT I think we should allow users to input text and images before the youtube link if they want. which means the tabs would be active anyway.
<img width="384" alt="Screenshot 2022-11-30 at 13 51 38" src="https://user-images.githubusercontent.com/94056434/207059607-aae01a00-aee8-405a-8766-d127bf3ba578.png">

**Comment by corinnewo** [comment]
I agree that you should be able to add images and text at any stage. But I think Gali's reasoning was that you must add in a video in the module, otherwise it is not a video module! I'll add to discuss at ideation.

**Comment by ChaykeeMor** [comment]
Could we not address that by preventing publishing when a video not included? Or even when you get to preview and say hey no video or something? Several times already, I have wanted to set up my screen with titles and images before placing my video.. but it's just an inconvenience. So not crucial to change or even prioritize.

**Comment by corinnewo** [comment]
Please change the video so that the user has access to all tabs. At the moment you can't design your screen until you have inserted the video. We want to give access to all tabs, and if you leave CONTENT tab without a video defined, then there should be an alert (@ChaykeeMor) and a design (@GaliShapira ) warning you that there is no video in the video module. 
This is similar to #3455 logic, where you are warned about empty content before you proceed.

**Comment by ChaykeeMor** [comment]
Oops! You forgot to insert a video link! Please make sure to add your video before moving to the next step.
